docs/src/1_Mechanics/01_PlayerGuide_Full.md

```diff
@@ -408,70 +437,79 @@ over. This ends the turn order, and characters can act independently of turn ord
 Characters do not discard their Target Cards after combat ends. This TC represents the
 character until the start of the next combat.
 
+If the Players are defeated in combat, the GM determines what happens next in the story.
+If the scenario was of life or death importance, the Player Characters may be killed
+outright. In less tense scenarios, the Player Characters may simply be left knocked out,
+imprisoned, or some other relevant circumstance.
+
 #### Status Conditions
```

Comment:
These were edited and then re-wrapped in the same commit, making it very difficult to see the changes